src/devtools/client/debugger/panel.js

```diff
@@ -42,8 +42,7 @@ export class DebuggerPanel {
   }
 
   async openConsoleAndEvaluate(input) {
-    const consolePanel = await this.toolbox.selectTool("console");
-    consolePanel.evaluateExpression(input);
+    this._actions.evaluateExpression(input);
```

Comment:
i didn't touch anything else in debugger because i'm doing each panel in separate pr's, i only touched those two lines because it was necessary to remove webconsole panel